src/components/visual-tooltip/VisualTooltip.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import Tooltip, { TooltipProps, TooltipTheme } from '../tooltip';
+import VisualTooltipContent from './VisualTooltipContent';
+
+import './VisualTooltip.scss';
+
+// We manually set "text" with our specific visual tooltip content.
+type OtherTooltipProps = Omit<TooltipProps, 'text'>;
+
+export type VisualTooltipProps = {
+    /** A React element to put the tooltip on */
+    children: React.ReactChild;
+    /** A string to be used in the tooltip's paragraph content */
+    content: string;
+    /** A React component representing the image used in the visual tooltip */
+    image: React.ReactNode;
+    /** A string to be used in the tooltip's title heading */
+    title: string;
+} & OtherTooltipProps;
+
+const VisualTooltip = ({ children, className, content, image, title, ...otherTooltipProps }: VisualTooltipProps) => {
```

Comment:
Great point! I was thinking `Visual` as like a visual graphic (not `visual` like sight). Perhaps something like `ImageTooltip` or `AssetTooltip` would be clearer?

Reply:
It seems you are building TitleImageContentTooltip if you want to be specific. On the first sight, AssetTooltip and ImageTooltip are partially accurate but still much better than visualTooltip. TitleImageContentTooltip is better?

Reply:
I like ImageTooltip since your comments mention an image used in it.  Are there non-image assets that we would want to use?  

The only asset that comes to mind that may not be categorized as an image is an icon..  but we may want a different component for icons since they tend to be smaller and would require different styling?

Reply:
Great feedback, you two! Thank you! I'm going to adjust to `ImageTooltip` (more succinct and still as clear IMHO).